service.subtitles.napisy24pl/addon.xml

```diff
@@ -1,19 +1,20 @@
-<?xml version="1.0" encoding="UTF-8" standalone="yes"?><addon id="service.subtitles.napisy24pl" name="Napisy24.pl" version="2.0.1" provider-name="CaTz">
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?><addon id="service.subtitles.napisy24pl" name="Napisy24.pl" version="2.0.2" provider-name="CaTz">
     <requires>
         <import addon="xbmc.python" version="2.14.0"/>
-		<import addon="script.module.beautifulsoup4" version="4.3.2"/>
+        <import addon="script.module.beautifulsoup4" version="4.3.2"/>
     </requires>
     <extension point="xbmc.subtitle.module" library="service.py"/>
     <extension point="xbmc.addon.metadata">
-		<summary lang="en">Napisy24.pl</summary>
-		<description lang="en">Search and Download subtitles from napisy24.pl.</description>
-		<summary lang="pl">Napisy24.pl</summary>
-		<description lang="pl">Szukaj i pobieraj napisy z serwisu napisy24.pl.</description>
-		<source>https://github.com/CaTzil/service.subtitles.napisy24pl</source>
-		<language/>
-		<email/>
-		<forum>http://forum.kodi.tv/showthread.php?tid=193829</forum>
-		<website/>
-		<license>GNU GENERAL PUBLIC LICENSE. Version 2, June 1991</license>
+        <summary lang="en">Napisy24.pl</summary>
+        <description lang="en">Search and Download subtitles from napisy24.pl.</description>
+        <summary lang="pl">Napisy24.pl</summary>
+        <description lang="pl">Szukaj i pobieraj napisy z serwisu napisy24.pl.</description>
+        <source>https://github.com/CaTzil/service.subtitles.napisy24pl</source>
+        <language>pl</language>
+        <email>x</email>
+        <forum>x</forum>
+        <website>x</website>
```

Comment:
empty email, forum and website tags have to be removed instead of tricking our addon checker

Reply:
right. @CaTzil please fix this and send an update

Reply:
https://github.com/xbmc/repo-scripts/pull/910